src/platform/forms/components/review/PreSubmitSection.jsx

```diff
@@ -64,18 +61,21 @@ export function PreSubmitSection(props) {
         <div>
           {preSubmit.notice}
           {preSubmit.required && (
-            <Checkbox
+            <va-checkbox
               required
               checked={checked}
-              onValueChange={value => setPreSubmit(preSubmit?.field, value)}
+              vaChange={value => setPreSubmit(preSubmit?.field, value)}
```

Comment:
Has this been tested? I'm just curious if the custom event `vaChange` is firing as expected with it being a vanilla web component (and not the React Binding).

See: https://design.va.gov/about/developers/using-web-components#custom-events

Reply:
I would have expected the custom event (vaChange ) handled more like this but maybe you've verified it already:

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/caregivers/components/FormFields/AddressWithAutofill.jsx#L132-L139

Reply:
I'll look into this